spec/ics-003-connection-semantics/README.md

```diff
@@ -134,6 +137,33 @@ function removeConnectionFromClient(
 }
 ```
 
+Two helper functions are defined to provide automatic application of `CommitmentPrefix`. These functions are recommended to be used
```

Comment:
Let's make them mandatory if we do it this way, otherwise the spec is ambiguous.

`CommitmentPrefix` can just be empty I presume.

Reply:
Yes `applyPrefix` can be a constant function.